libclamav/yara_arena.h

```diff
@@ -80,7 +80,7 @@ void* yr_arena_base_address(
 void* yr_arena_next_address(
     YR_ARENA* arena,
     void* address,
-    int offset);
+    size_t offset);
```

Comment:
Looks like the upstream libyara project made this and other related changes. Seems okay to me. https://github.com/VirusTotal/yara/commit/f502bdf4903545cfc75dfe15bed650138da53a81

I don't mind making this change, but I would note this will all get replaced when we finally get around to upgrading our yara-derived sources. Related discussion in: https://github.com/Cisco-Talos/clamav/issues/283